docs/rules.md

```diff
@@ -10,7 +10,8 @@ We should try and list out all the validation rules we know of by file type. E.g
 | Done? | MsgType | Rule name | Addon type | Description | File Type | Source ref | Old Code | New Code |
 | ----- | ------- | --------- | ---------- | ----------- | --------- | ---------- | -------- | -------- |
 | :x: | warning | dangerous_contract | | Dangerous XPCOM contract ID |  | [testcases/javascript/actions.py](https://github.com/mozilla/amo-validator/blob/master/validator/testcases/javascript/actions.py)|
-| :x: | warning |  banned_identifier | | Banned or deprecated JavaScript Identifier | | | | |
+| :white_check_mark: | warning |  banned_identifier | | Banned or deprecated JavaScript Identifier | | | | BANNED_NEWTHREAD |
```

Comment:
>  Oh, just looked in the code below, seems there is an old code. Should it be added in here?

Yeah I've not done that yet (I presume you're referring to the tuple stuff which is legacyCode?)